client-src/elements/form-field-specs.js

```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
     help_text: html`Link(s) to TAG review(s), or explanation why this is
                 not needed.`,
+    extra_help: html`
+    <p>
+      The <a target="_blank" href="https://www.w3.org/2001/tag/">W3C Technical Architecture Group</a> (TAG) 
+      is a special working group of the W3C that consists of a few appointed and elected members, 
+      all of whom are experienced members of the web standards community. 
+      The Blink launch process has a formal requirement for requesting a 
+      <a target="_blank" href="https://github.com/w3ctag/design-reviews">TAG review</a> for all features. 
+      The review happens publicly on a GitHub issue.
+    </p>
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature. 
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> at the I2E stage (right after is also fine). 
+      A large number of Intents to Ship are delayed because a TAG review was only recently filed 
+      and engagement from the TAG can take multiple weeks to multiple months.
+    </p>
+    <p>  
+      Note that only the request for TAG review is strictly required by the process. 
+      We don't block shipping on a response from the TAG, but a reasonable effort must be made 
+      to obtain their review with enough time for them to give feedback. 
+      There may be some work involved in preparing your feature for review (see the 
+        <a target="_blank" href=
+          "https://github.com/w3ctag/design-reviews/blob/main/.github/ISSUE_TEMPLATE/010-specification-review.md"
+        >submission template fields</a>).
```

Comment:
I think I'd move this chunk up to before the "A large number of Intents to Ship are delayed" sentence, and maybe put the paragraph break between them. It seems more connected to the "file the review" sentence than the "consequences of not filing the review".

Reply:
agreed